src/components/editMeasure/testCases/components/routes/qdm/TestCaseRoutes.tsx

```diff
@@ -209,7 +221,7 @@ const TestCaseRoutes = () => {
                   <TestCaseLandingQdm
                     errors={cqmMeasureErrors}
                     setErrors={setCqmMeasureErrors}
-                    setWarnings={setImportWarnings}
+                    setImportWarnings={setImportWarnings}
```

Comment:
This should also need setWarnings, The 2 different routes should land user on the same page. and will have same funtionality.